managed/src/main/java/com/yugabyte/yw/commissioner/tasks/subtasks/cloud/CloudAccessKeySetup.java

```diff
@@ -53,6 +53,8 @@ public void run() {
         taskParams().sshPrivateKeyContent == null || taskParams().sshPrivateKeyContent.isEmpty() ||
         taskParams().sshUser == null || taskParams().sshUser.isEmpty()) {
       String sanitizedProviderName = getProvider().name.replaceAll("\\s+", "-").toLowerCase();
+      sanitizedProviderName = String.format(
+        "%s%s", sanitizedProviderName, taskParams().providerUUID);
```

Comment:
Do we have a unit test for this? Also, I would add an hyphen. Would also make the modification in the next line, rather than update the sanitized name.

Reply:
I prefer underscore between name and uuid

Reply:
We don't have UT for this file itself. We can create one but since this is just a small changes so just manually tested i think this should be fine for now. 

Adding underscore  between name and uuid make sense
yb-admin-aws_87db5f86-b61a-42a7-9eee-9d600c18bb04-key.

I agree, we should add provideUUID in the next line instead of modifying the existing sanitizedProviderName.

Thanks!

Reply:
Sounds good! Will Approve after the changes.